sections/staking/components/BurnTab/BurnTab.tsx

```diff
@@ -187,7 +187,6 @@ const BurnTab: React.FC = () => {
 								setTransactionState(Transaction.SUCCESS);
 							},
 						});
-						onBurnChange('0');
```

Comment:
@clementbalestrat if we don't reset the input, when does it get configured to 0?